pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -62,6 +64,32 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 		if err != nil {
 			return nil, err
 		}
+
+		// XXX Do we always want to create this policy or should we do this lazily
+		// and create when requested
+		err = s.createProxyProtocolPolicy(loadBalancerName)
```

Comment:
I would prefer lazy creation personally.  But I don't think it's a blocker.

One way to make lazy creation easy - just try to create when you need it, and if you get an "already exists" error ignore it (still raise any other errors!)  If it turns out this hits a different rate limit then we can deal with that..

The other advantage of doing lazy creation is that if someone upgrades from k8s 1.2 to 1.3 (or whatever) and gets your code, but has pre-existing ELBs, we won't assume they were created.  Or if someone sees that the policy is unused and deletes it, we will recover from that.